frontend/src/lib/taxonomy.tsx

```diff
@@ -990,6 +990,10 @@ export const CORE_FILTER_DEFINITIONS_BY_GROUP = {
             label: 'ttclid',
             description: 'TikTok Click ID',
         },
+        $is_identified: {
+            label: 'Is Identified',
+            description: 'Has the user called identify()',
```

Comment:
```suggestion
            description: 'When the person was identified',
```
I think "user" is confusing in this context, since we call that our customer's users for other properties